test/ImportCommand.js

```diff
@@ -69,6 +75,28 @@ describe("ImportCommand", () => {
       }));
     });
 
+    it("works with --max-buffer", (done) => {
```

Comment:
That being said, it is clearly important to ensure that `ImportCommand` is mixing `this.execOpts` into `this.externalExecOpts` correctly. (no mocking of `ChildProcessUtilities.execSync()` required)

```js
const ONE_HUNDRED_MEGABYTES = 1000 * 1000 * 100;
const importCommand = new ImportCommand([externalDir], {
  maxBuffer: ONE_HUNDRED_MEGABYTES,
}, testDir);

// runCommand() then...
expect(importCommand.execOpts).toHaveProperty("maxBuffer", ONE_HUNDRED_MEGABYTES);
expect(importCommand.externalExecOpts).toHaveProperty("maxBuffer", ONE_HUNDRED_MEGABYTES);
```